ParkingZone/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -111,4 +112,34 @@ public IActionResult Details(long id)
         DetailsVM detailsVM = new(existSlot);
         return View(detailsVM);
     }
+
+    [HttpGet]
+    public IActionResult Delete(long? id)
+    {
+        if (id == null)
```

Comment:
I think you do not need to check whether id null cuz if id null it is impossible to enter view